internal/nginx/config/http/config.go

```diff
@@ -65,3 +73,16 @@ type SplitClientDistribution struct {
 	Percent string
 	Value   string
 }
+
+// Map defines an NGINX map.
+type Map struct {
+	Source     string
+	Variable   string
+	Parameters []Parameter
+}
+
+// Parameter defines a Value and Result pair in a Map.
+type Parameter struct {
```

Comment:
Can we rename it to MapParameter? Because Parameter name is very broad, can be a Parameter to anything